tests/013_http_import.ts

```diff
@@ -0,0 +1,2 @@
+import "http://localhost:8000/script.ts"
+import "http://localhost:8000/this/is/a/directory/script.ts"
```

Comment:
Please use the test from the prototype instead of this:
https://github.com/denoland/deno/blob/golang/testdata/006_url_imports.ts